### ISSUE TITLE
[Carbon web components]: investigate migration of `carbon-web-components` package to monorepo

### ISSUE THREAD

**Issue by kennylam** [carbon-design-system/carbon-web-components#974]
### The problem

`carbon-web-components` package needs to be migrated into the monorepo.

This includes:
- re-target Jenkins jobs to Carbon for IBM.com repo
- updating build scripts to use internal vendor packages
- consider consolidating all deploy preview links into one GH comment (`vue`, `ng`, etc)

### The solution

Do all the stuff above.

### Application/website

Carbon for IBM.com

### Business priority

Medium Priority = upcoming release but is not pressing

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-web-components/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-web-components/issues) for duplicate issues

**Comment by emyarod** [comment]
@abdonrd Yes, the migration will be to [carbon-design-system/carbon-for-ibm-dotcom](https://github.com/carbon-design-system/carbon-for-ibm-dotcom), and the package will still be published independently

**Comment by abdonrd** [comment]
No news about this? Thanks!
